guides/source/5_0_release_notes.md

```diff
@@ -55,7 +55,7 @@ information.
 ### API Applications
 
 Rails can now be used to create slimmed down API only applications.
-This is useful for creating and serving APIs similar to [Twitter](https://dev.twitter.com) or [GitHub](https://developer.github.com) API,
+This is useful for creating and serving APIs similar to [X](https://dev.x.com) or [GitHub](https://developer.github.com) API,
```

Comment:
https://developer.x.com/

Reply:
Thanks for pointing out. Changing right now.